docs/src/test/scala/docs/http/scaladsl/server/PlayRoutesComparisonSpec.scala

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (C) 2009-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package docs.http.scaladsl.server
+
+import akka.http.scaladsl.common.ToNameReceptacleEnhancements
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server._
+import akka.http.scaladsl.testkit.ScalatestRouteTest
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+
+class PlayRoutesComparisonSpec extends AnyWordSpec with Matchers with ScalatestRouteTest {
+
+  object Clients {
+    def list(): String = "clientA,clientB,clientC"
+
+    def get(id: Long): String = "clientB"
+  }
+
+  "Play examples" should {
+
+    "show a fixed URL" in {
+      val clientsAll: Route =
+        // #fixed
+        (get & path("clients" / "all")) {
```

Comment:
While you could write it that way, I think we should recommend a different style in our new best practices guide. There, we would prefer matching on path first and then on method. But in general, those might not be in the same line of code. We could still show them in the same line of code but would then point back to that guide about how you would do it more generally.

Reply:
Hmm, that's a good idea. If, in the very fisrt example we introduced:

```
// Play routes file
GET				/clients/all    controllers.getAll()

// Akka-HTTP
(get & path("clients" / "all")) {
  ...
}

// idiomatic Akka-HTTP
path("clients" / "all"){
  get {
    ...
  }
}
```

So users can see it's possible to do a more direct conversion but we specify, early on in the page, that the idiomatic Akka-HTTP approach is to match on the path before matching on the method.

Reply:
I'd like to keep the focus on the direct comparison and link to the style guide.

Reply:
Ah, good idea! Introducing the style guide early is a better approach.